docs/migrations/migrations.md

```diff
@@ -0,0 +1,232 @@
+
+It can help to approach every non-trivial upgrade as a relaunch.
+It is usually much easier to scale down expectations than up.
+
+```{note}
+Arguing for a required switch of a Python 2.7 installation of Plone 5.2.6 to Python 3 (while staying on 5.2.6) is hard without taking the opportunity to change the mobile navigation.
+
+Suddenly it's a relaunch and everyone is excited.
+```
+
+
+(migrations-major-changes-label)=
+
+## Dealing with major changes in Plone
+
+In the history of Plone so far, there were three major changes:
+
+* Plone 5.0: Dexterity replaces Archetypes
+* Plone 5.2: Support for Python 3
+* Plone 6.0: Volto as the new default frontend
+
+Each of these major changes requires special treatment during an upgrade that makes upgrading much more complex.
+
+
+(migrations-plone-5.0-dexterity-replaces-archetypes-label)=
+
+### Plone 5.0: Dexterity replaces Archetypes
+
+With Plone 5.0 the default framework for content types switched from Archetypes to Dexterity.
+
+Up through Plone 5.2.x, there is a built-in migration from Archetypes to Dexterity, but it only supports Python 2.
+See [Migration](https://pypi.org/project/plone.app.contenttypes/2.2.3/#migration) in the latest stable release of `plone.app.contenttypes` for details on the migration of custom and default content types to Dexterity.
+
+Using [collective.exportimport](https://pypi.org/project/collective.exportimport/) you can export Archetypes content and import it as Dexterity content.
+
+
+(migrations-plone-5.2-support-for-python-3-label)=
+
+### Plone 5.2: Support for Python 3
+
+Plone 5.2 added support for Python 3, while Plone 6.0 dropped support for Python 2.
+This means that you can use Plone 5.2 to upgrade to Python 3.
+
+This requires that you run Plone in Python 3 and only use code that supports Python 3.
+It also requires that you migrate the database in a separate step from Python 2 to 3 while Plone is not running.
+
+See the upgrade guides {doc}`plone6docs:backend/upgrading/version-specific-migration/upgrade-to-python3` and {doc}`plone6docs:backend/upgrading/version-specific-migration/upgrade-zodb-to-python3` for detailed information on these steps.
+
+Using [`collective.exportimport`](https://pypi.org/project/collective.exportimport/), you can export content from Python 2 and import it in Python 3.
+
+
+(migrations-plone-6.0-volto-as-new-frontend-label)=
+
+### Plone 6.0: Volto as new frontend
+
+Plone 6.0 comes with a new default frontend called {term}`Volto`.
+It is written in React, and expects some subtle but important changes.
+
+See {doc}`plone6docs:backend/upgrading/version-specific-migration/migrate-to-volto` for the specific migration steps.
+
+
+### Transition periods for major changes
+
+To make these major changes easier to manage, Plone provides a transition for all of these scenarios:
+
+* You can use Dexterity starting with Plone 4.0, and use Archetypes until Plone 5.2 (as long as you use Python 2).
```

Comment:
Do note that dexterity   for Plone was a 1.x version, a lot of bugs and inconsistentpcies were fixed in dexterity 2.x and higher included with Plone 5.x versions.

Reply:
I'll would not include that - too much detail. I'd also have to discuss issues with AT in 5.2 on python2.